.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+
+node_js:
+  - "6"
+
+os:
+  - linux
+  - osx
+
+script:
+  - npm run rebel
+
+after_success:
+  - sh travis/set_tags.sh
+
+before_deploy: bash -e travis/before-deploy.sh
+
+deploy:
+  provider: releases
+  file_glob: true
+  file: _build/rebel-*.zip
```

Comment:
we discussed not having `.zip` files and instead uploading the exact directory including artifacts to a github tag.